.htaccess

```diff
@@ -12,6 +12,7 @@ redirect 301 /doc/sbt/run.html {{ BASE_PATH }}/doc/project/building.html
 redirect 301 /doc/sbt/depending.html {{ BASE_PATH }}/doc/project/dependencies.html
 redirect 301 /doc/sbt/js-envs.html {{ BASE_PATH }}/doc/project/js-environments.html
 redirect 301 /doc/sbt/cross-building.html {{ BASE_PATH }}/doc/project/cross-build.html
+redirect 301 /doc/sbt/testing.html {{ BASE_PATH }}/doc/project/testing.html
```

Comment:
I don't think this is needed. IIUC, these are just so that links to the old site still work (a comment would have been useful indeed).